.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "8"
   - "10"
   - "12"
   - "13"
+  - "14"
```

Comment:
should we remove .travis.yml completely when introducing github actions?

Reply:
I'd say yes.

Reply:
Question is, should I do it in this PR? Or do you want to do it afterwards. Maybe run both pipelines once and if both are successful, remove the travis PL.

Reply:
Oh and also we should change the badge in the readme to point to the correct pipeline.

Reply:
I've removed it and cherry picked the commit with the badge from your master into this.